jenkins/databricks/build.sh

```diff
@@ -103,36 +103,5 @@ mvn -B install:install-file \
 
 mvn -B -P${BUILD_PROFILES} clean package -DskipTests
 
-# Copy so we pick up new built jar and latest CuDF jar. Note that the jar names have to be
```

Comment:
Move integration tests scripts into 'test.sh', to make it common for DB build & IT pipelines